### DISCUSSION TITLE
Allow saving all settings

### DISCUSSION THREAD

**Discussion by burningT** [TechbeeAT/jtxBoard#1332] [Ideas]
Currently presets can be saved and retored. A lot of valuable settings can be made at other places. However, there seems to be no way to save most of them at present. Wouldn't it be helpful to allow saving and restoring all settings?

**Comment by patrickunterwegs** [comment]
Hi @burningT , in fact, I have never thought about this, but yes, it makes sense. I might just include it in the export/import of presets. What do you think?

**Reply by burningT** [reply]
That would be a good thing. Although I would recommend rethinking the placement. I guess most would expect the button in that spot to store exactly what it stores now. The feature might remain unnoticed. Maybe placing some button next to the settings is more intuitive?

**Reply by patrickunterwegs** [reply]
Hi @burningT , sorry, I forgot to follow up on this!
Yes you're right, I am thinking about adding another menu on the top right in the settings menu with an option to export the settings (similar to the presets screen) =)
